### PR TITLE
sdl2 2.0.10

### PR DESCRIPTION
**Pull request by illume**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/sdl2.rb [hunk 1]
```diff
@@ -4,18 +4,8 @@ class Sdl2 < Formula
   revision 1
```
Comment (Moisan) [inline Formula/sdl2.rb]: This line should be removed.

**Review by Moisan — CHANGES_REQUESTED** [review]

File: Formula/sdl2.rb (hunk 1 above)
Comment (illume) [reply Formula/sdl2.rb]: Thanks. `revision 1` was removed.

**Comment by illume** [comment]
I closed the PR, and opened it again... which seems to have caused the jenkins build to be rerun.

*fingers crossed*

**Comment by fxcoudert** [comment]
Thanks @illume

**Pull request closed without merging**